ci/upload_to_maven.sh

```diff
@@ -12,4 +12,4 @@ export PROJECT_VERSION=$(./gradlew properties -q | grep "version:" | awk '{print
 if [[ ! $(echo "${PROJECT_VERSION}" | grep "SNAPSHOT") ]]; then
   ./gradlew closeAndReleaseRepository
 fi
-echo "Update to maven completed"
+#echo "Update to maven completed"
```

Comment:
Remove it entirely, no need to leave dead code.

Reply:
done